docker/test/integration/minifi/core/NifiContainer.py

```diff
@@ -54,11 +55,12 @@ def deploy(self):
         logging.info('Creating and running nifi docker container...')
         self.__create_config()
         self.client.containers.run(
-            self.image_store.get_image(self.get_engine()),
+            "apache/nifi:" + NifiContainer.NIFI_VERSION,
```

Comment:
was this change left in by accident? all other containers seem to use the former version

Reply:
This change was on purpose as the newest image declares the config directory as `VOLUME` so it cannot be changed in build only in runtime which is done that way now. Because of this there is no need to do anything in build time we can use the docker image as it is. Not all containers need a changed base image, for example the Azure image is done this way too.